src/surfacehop.F90

```
@@ -31,16 +30,20 @@ module mod_sh
    public :: nacx, nacy, nacz
 
    ! Initial electronic state
-   integer :: istate_init = 1
+   integer :: istate_init = -1
```

I have changed the default to invalid number so that users don't forget to set initial state.